examples/nl-core-DOSScore-01.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-DOSScore"/>
+   </meta>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="160591000146109"/>
+         <display value="'Delirium observation screening'-beoordelingsschaal"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <effectiveDateTime value="2016-12-19T10:35:00+01:00"/>
+   <valueInteger value="4"/>
+   <note>
+      <text value="Patiënt is mogelijk delirant."/>
+   </note>
+   <component>
+      <code>
+         <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1"/>
```

Comment:
There is a error about the system value:

  -  error at Observation.component[0].code.coding[0] (26, 18):
     Unknown Code urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1#18007006 in urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1 for 'urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0.1#18007006'

This need to be fixed in the Validation tool